### PR TITLE
docs: tidy up quickstart & other docs polish

### PR DESCRIPTION
**Pull request by leeederek**
## Description:
Quickstart and some other docs polish based on feedback this week from Kevin and our quickstart feedback sessions

- rename "what is kurtosis" to "why we built kurtosis"
- Re-titled "Why we built Kurtosis?" to "Why we built Kurtosis"
- Re-formatted our conclusion in the quickstart to recap the learnings, quickly mention other docs, show users the examples, and then finally cap it all off with feedback _while keeping everything short and sweet_
- Made "setup" in our quickstart more obvious
- Set clearer expectations upfront in our quickstart that we expect users to code-along and that they will be running only free, public data/software

## Is this change user facing?
YES

### CODE REVIEW

**Comment by leeederek** [comment]
- Added a call-back by linking the error we encounter (when adding our API) to when we were explaining multi-phase runs 
- Removed a comment in a TIP box that suggests users skip the "review" section entirely if they want only action. We definitely want users to be reading and learning from the "review" section. 
- Removed the detailed future reference example to reduce cognitive load on the user
- ~~Throughout the quickstart, swapped mentions of "you" to "we" or "our" to make the quickstart feel more collaborative and team-work-esque.~~ Reverted this based on feedback from Galen
- Added a tip box to explain that a Locator != URL, but they're similar and is what Kurtosis uses to link remotely hosted packages together 
- In the "Add some data" section, moved the section about `kurtosis.yml` to higher up in the docs to match the way our review section recaps what the user does
- Added a tip that the arg after `--enclave` is the name of the enclave to use, and if one doesn't exist, create a new enclave with that given arg as the name of the enclave
- Changed the hyperlink in the quickstart introduction to the home page of our docs rather than our website 
- Reduced new tool jargon when it comes to our first introduction about Starlark. Specifically, removed mentions of Bazel and JARs and link out to the new Starlark docs instead

File: docs/docs/home.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ hide_table_of_contents: true
 ## What is Kurtosis?
 [Kurtosis](https://www.kurtosis.com) is a composable build system for multi-container test environments. Kurtosis makes it easier for developers to set up test environments that require dynamic setup logic (e.g. passing IPs or runtime-generated data between services) or programmatic data seeding.
 
+Go [here](./explanations/why-we-built-kurtosis.md) to learn more the inspiration behind starting Kurtosis.
```
Comment (galenmarchetti) [inline docs/docs/home.md]:
```suggestion
Go [here](./explanations/why-we-built-kurtosis.md) to learn more what inspired us to build Kurtosis.
```

File: docs/docs/quickstart.md [hunk 2]
```diff
@@ -7,15 +7,17 @@ slug: /quickstart
...
+For a quick read on what Kurtosis is and what problems Kurtosis aims to solve, our [introduction page][homepage] will be a great starting point, alongside our [motivations behind starting Kurtosis][why-we-built-kurtosis-explanation].
 
 :::tip What You'll Do
 
 - Start a containerized Postgres database in Kurtosis
-- Seed your database with test data using task sequencing
-- Connect an API server to your database using dynamic service dependencies
-- Parameterize your application setup in order to automate loading data into your API
-:::
+- Seed our database with test data using task sequencing
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
- Seed your database with test data using task sequencing
```

File: docs/docs/quickstart.md [hunk 3]
```diff
@@ -7,15 +7,17 @@ slug: /quickstart
...
+- Connect an API server to our database using dynamic service dependencies
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
- Connect an API server to your database using dynamic service dependencies
```

File: docs/docs/quickstart.md [hunk 4]
```diff
@@ -7,15 +7,17 @@ slug: /quickstart
...
+- Parameterize our application setup in order to automate loading data into our API
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
- Parameterize your application setup in order to automate loading data into your API
```

File: docs/docs/quickstart.md [hunk 5]
```diff
@@ -7,15 +7,17 @@ slug: /quickstart
...
+:::a
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
:::
```

Comment (galenmarchetti) [reply docs/docs/quickstart.md]: I'm guessing this a is a typo? lmk if wrong

File: docs/docs/quickstart.md [hunk 6]
```diff
@@ -70,7 +74,7 @@ Finally, [run][kurtosis-run-reference] the script (we'll explain enclaves in the
 kurtosis run --enclave quickstart main.star
 ```
 
-Kurtosis will work for a bit, and then deliver you results:
+Kurtosis will work for a bit, and then deliver us the following result:
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
Kurtosis will work for a bit, and then deliver you the following result:
```

File: docs/docs/quickstart.md [hunk 7]
```diff
@@ -99,16 +103,15 @@ Congratulations - you've written your first Kurtosis code!
 
 ### Review: Hello, World
 :::info
-We'll use these "Review" sections to explain what happened in the section. If you just want the action, feel free to skip them.
+You'll use these "Review" sections to explain what happened in the section. 
 :::
 
-In this section, we created a `main.star` file that prints `Hello, world`. The `.star` extension corresponds to [the Starlark language developed at Google][starlark-github-repo], a dialect of Python for configuring the [Bazel build system][bazel-github]. [Kurtosis uses Starlark for the same purpose of configuring builds][starlark-explanation], except that we're building a distributed application rather than binaries or JARs.
+In this section, you created a `main.star` file that simply told Kurtosis to print `Hello, world`. The `.star` extension corresponds to [Starlark][starlark-explanation], which is also used by Google and Meta as a build system configuration language.
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
In this section, you created a `main.star` file that simply told Kurtosis to print `Hello, world`. **The `.star` extension corresponds to [Starlark][starlark-explanation], a Python dialect also used by Google and Meta for configuring build systems.**
```

Comment (galenmarchetti) [reply docs/docs/quickstart.md]: bolded this because people keep overlooking it, and changed the wording a bit

File: docs/docs/quickstart.md [hunk 8]
```diff
@@ -199,7 +204,13 @@ By contrast, Kurtosis Starlark scripts can use data as a first-class primitive a
 
 Let's see it in action, and we'll explain what's happening afterwards.
 
-First, replace your `main.star` with the following:
+First, in your working directory (kurtosis-quickstart) next to your `main.star` file, create a file called `kurtosis.yml` with the following contents:
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
First, in your working directory (`kurtosis-quickstart`), next to your `main.star` file, create a file called `kurtosis.yml` with the following contents:
```

File: docs/docs/quickstart.md [hunk 9]
```diff
@@ -385,9 +390,13 @@ data_package_module = import_module("github.com/kurtosis-tech/awesome-kurtosis/d
...
 
-This external Kurtosis package, named ["data-package"][data-package-example] contains the seed data for our Postgres instance that we [referenced earlier](#add-some-data) as a `.tar` file.
+This external Kurtosis package, named ["data-package"][data-package-example] contains the seed data for your Postgres instance that we [referenced earlier](#add-some-data) as a `.tar` file.
+
+:::info
+Special note here that we used a locator to import an external package from your [`awesome-kurtosis` repository][awesome-kurtosis-repo] and _not_ a regular URL. Learn more about how they differ [here][locators-reference].
+:::
 
-#### We imported seed data into our Kurtosis package
+#### We imported seed data into your Kurtosis package
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
#### You imported seed data into your Kurtosis package
```

File: docs/docs/quickstart.md [hunk 10]
```diff
@@ -405,8 +414,8 @@ def run(plan, args):
 
 A [files artifact][files-artifacts-reference] is Kurtosis' first-class data primitive and is a TGZ of arbitrary files living inside an enclave. So long as a files artifact exists, Kurtosis knows how to mount its contents on a service.  
 
-#### We mounted and seeded the data into our Postgres instance
-Next, we mounted the seed data, stored in our enclave now as a files artifact, into our Postgres instance using the `ServiceConfig.files` option:
+#### We mounted and seeded the data into your Postgres instance
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
#### You mounted and seeded the data into your Postgres instance
```

File: docs/docs/quickstart.md [hunk 11]
```diff
@@ -590,7 +604,7 @@ kurtosis service logs quickstart api
 postgrest: thread killed
 ```
 
-Looking back to our Starlark code, we can see the problem: we're creating the Postgres database with a user called `app_user`, but we're telling PostgREST to try and connect through a user called `postgres`:
+Looking back to your Starlark code, we can see the problem: we're creating the Postgres database with a user called `app_user`, but we're telling PostgREST to try and connect through a user called `postgres`:
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
Looking back to your Starlark code, you can see the problem: it's creating the Postgres database with a user called `app_user`, but it's telling PostgREST to try and connect through a user called `postgres`:
```

File: docs/docs/quickstart.md [hunk 12]
```diff
@@ -684,11 +698,11 @@ api = plan.add_service(
 
 ...Kurtosis simply swapped in the correct Postgres container Execution-time values. While future references take some getting used to, [we've found the feedback loop speedup to be very worth it][why-multi-phase-runs-explanation].
 
-**What you've just seen is Kurtosis' powerful ability to data generated at runtime to set up service dependencies in multi-container test environments. We also saw how seamless it was to to run on-box CLI commands on a container.**
+**What we've just seen is Kurtosis' powerful ability to data generated at runtime to set up service dependencies in multi-container test environments. We also saw how seamless it was to to run on-box CLI commands on a container.**
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
**What you've just seen is Kurtosis' powerful ability to data generated at runtime to set up service dependencies in multi-container test environments. You also saw how seamless it was to to run on-box CLI commands on a container.**
```

File: docs/docs/quickstart.md [hunk 13]
```diff
@@ -684,11 +698,11 @@ api = plan.add_service(
...
 
 Modifying data
 --------------
-Now that we have an API, we should be able to interact with the data.
+Now that you have an API, we should be able to interact with the data.
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
Now that you have an API, you should be able to interact with the data.
```

File: docs/docs/quickstart.md [hunk 14]
```diff
@@ -879,7 +893,7 @@ At a higher level, Kurtosis automatically deserialized the `[{"first_name":"Kevi
 def run(plan, args):
 ```
 
-**This section showed us how we can interact with the test environment we set up and also how to parametrize our distributed application for others to re-use.**
+**This section showed us how we can interact with the test environment we set up and also how to parametrize your distributed application for others to re-use.**
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
**This section showed how to interact with your test environment, and also how to parametrize it for others to easily modify and re-use.**
```

File: docs/docs/quickstart.md [hunk 15]
```diff
@@ -941,19 +955,22 @@ Let's review. In this tutorial you have:
 
 - Started a Postgres database in an ephemeral, isolated test environment
 - Seeded your database by importing an external Starlark package from the internet
-- Set up an API server for our database and gracefully handled dynamically generated dependency data
+- Set up an API server for your database and gracefully handled dynamically generated dependency data
 - Inserted & queried data via the API
 - Parameterized data insertion for future use
 
-Thank you for trying our quickstart. We hope you enjoyed it. 
+This was still just an introduction to Kurtosis. Feel read about these topics (and more) further in our docs where you can learn [what Kurtosis is][homepage], understand the [architecture][architecture-explanation], and hear our [inspiration for starting Kurtosis][why-we-built-kurtosis-explanation].
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
This was still just an introduction to Kurtosis. To dig deeper, visit other sections of our docs where you can read about [what Kurtosis is][homepage], understand the [architecture][architecture-explanation], and hear our [inspiration for starting Kurtosis][why-we-built-kurtosis-explanation]. 
```

File: docs/docs/quickstart.md [hunk 16]
```diff
@@ -941,19 +955,22 @@ Let's review. In this tutorial you have:
...
 - Inserted & queried data via the API
 - Parameterized data insertion for future use
 
-Thank you for trying our quickstart. We hope you enjoyed it. 
+This was still just an introduction to Kurtosis. Feel read about these topics (and more) further in our docs where you can learn [what Kurtosis is][homepage], understand the [architecture][architecture-explanation], and hear our [inspiration for starting Kurtosis][why-we-built-kurtosis-explanation]. 
 
-This was still just an intro to Kurtosis. To see examples that you can easily modify to be relevant to you, [check out our `awesome-kurtosis` repo][awesome-kurtosis-repo]. To explore real-scale Kurtosis packages delivering value, see the [Ethereum package][ethereum-package], [Waku package][waku-package], or [NEAR package][near-package] as well as our list of [Kurtosis-provided packages being used in production][kurtosis-managed-packages] and [Kurtosis packages in the wild][wild-kurtosis-packages].
+To keep exploring the world of Kurtosis beyond our docs, we encourage you to check out our [`awesome-kurtosis` repository][awesome-kurtosis-repo], where you will find real-world examples of how Kurtosis can be used, including:
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
To learn more about how Kurtosis is used, we encourage you to check out our [`awesome-kurtosis` repository][awesome-kurtosis-repo], where you will find real-world examples of Kurtosis in action, including:
```

File: docs/docs/quickstart.md [hunk 17]
```diff
@@ -941,19 +955,22 @@ Let's review. In this tutorial you have:
...
+- The [Ethereum package][ethereum-package], used by the Ethereum Foundation, which can be used to set up local testnets 
+- A powerful, paramterized package for standing up a local [Cassandra cluster with Grafana and Prometheus][cassandra-package-example] out-of-the-box
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
- A parameterized package for standing up an [n-node Cassandra cluster with Grafana and Prometheus][cassandra-package-example] out-of-the-box
```

File: docs/docs/quickstart.md [hunk 18]
```diff
@@ -941,19 +955,22 @@ Let's review. In this tutorial you have:
...
 
-Finally, we'd love to hear from you and engage with our users. Please don't hesitate to:
-- Share with us what went well, and what didn't, using `kurtosis feedback` to file an issue in our [Github](https://github.com/kurtosis-tech/kurtosis/issues/new/choose), to [email us](mailto:feedback@kurtosistech.com), or to [chat with our cofounder, Kevin](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding)
-- [Star us on Github](https://github.com/kurtosis-tech/kurtosis) (this helps a lot!)
-- [Join our Discord](https://discord.com/channels/783719264308953108/783719264308953111) (also available with the `kurtosis discord` CLI command)
-- [Reach out to us on Twitter](https://twitter.com/KurtosisTech) (also available with the `kurtosis twitter` CLI command)
+Finally, we'd love to hear from you and engage with our users. Please don't hesitate to share with us what went well, and what didn't, using `kurtosis feedback` to file an issue in our [Github](https://github.com/kurtosis-tech/kurtosis/issues/new/choose), to [email us](mailto:feedback@kurtosistech.com), or to [chat with our cofounder, Kevin](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding).
```
Comment (galenmarchetti) [inline docs/docs/quickstart.md]:
```suggestion
Finally, we'd love to hear from you. Please don't hesitate to share with us what went well, and what didn't, using `kurtosis feedback` to file an issue in our [Github](https://github.com/kurtosis-tech/kurtosis/issues/new/choose), to [email us](mailto:feedback@kurtosistech.com), or to [chat with our cofounder, Kevin](https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding).
```

**Review by galenmarchetti — CHANGES_REQUESTED** [review]

**Review by galenmarchetti — APPROVED** [review]

File: docs/docs/quickstart.md (hunk 7 above)
Comment (leeederek) [reply docs/docs/quickstart.md]: Spoke to Galen offline, not going to bold this section.

**Pull request merged**